src/fetch/period_slippage.py

```diff
@@ -5,16 +5,43 @@
 from src.dune_analytics import DuneAnalytics, QueryParameter
 from src.file_io import File
 from src.models import Network
+from src.token_list import HOSTED_ALLOWED_BUFFER_TRADING_TOKEN_LIST_URL, get_trusted_tokens_from_url
+
+
+def generate_sql_query_for_allowed_token_list(token_list) -> str:
```

Comment:
I would like to see a test for this method.

Reply:
How do we know this returns a valid query? What if the token list is empty, has only one element, multiple elements. It is good to test this so we can be confident it always yields a valid query (so we don't ever need to assume this is the reason for a bug).